### ISSUE TITLE
RSS2IRC flood the channel

### ISSUE THREAD

**Issue by YamashitaRen** [areeb-beigh/RSS2IRC#6]
RSS2IRC print the latest feed, every 2 minutes. It should print it only once, when a new entry is created.

From the chan : 
```
[11:30:54] --> [RSS]Cassandre (~Cassandre@285D2ECA.37E883BE.45B6C803.IP) a rejoint #Genshiken
[11:30:54] <[RSS]Cassandre> [RSS]Cassandre Now Online - Checking latest feed - !feed help to view commands
[11:30:54] <[RSS]Cassandre> [GSK]Yolocr-OCR | https://www.nyaa.se/?page=download&tid=897339
[11:32:59] <[RSS]Cassandre> [RSS]Cassandre Now Online - Checking latest feed - !feed help to view commands
[11:32:59] <[RSS]Cassandre> [GSK]Yolocr-OCR | https://www.nyaa.se/?page=download&tid=897339
[11:33:28] <YamashitaRen> k
[11:33:38] <-- [RSS]Cassandre (~Cassandre@285D2ECA.37E883BE.45B6C803.IP) est parti (Remote host closed the connection)
```

From the console : 
```
[':[RSS]Cassandre!~Cassandre@285D2ECA.37E883BE.45B6C803.IP', 'JOIN', ':#Genshiken']
Joined to: set()
Kicked from: set()
[':irc.rizon.systems', '332', '[RSS]Cassandre', '#Genshiken', ':Releases', ':', 'card', 'captor', 'Sakura??', '|Macross', 'Delta', 'complete', '|', 'Tricksters', '17/24', '|', 'Kab$neri', 'complete', '(BD)', '|', 'Genshiken', 'Nidaime', '04/13', 'on', 'need', 'un', 'enco', '|', 'http://www.genshiken.tk/', '|', 'USA', 'MAKE', 'AMERICA', 'GREAT', 'AGAIN', '|', $YoloCR', ':', 'http://v.gd/jC23ic']
[':irc.rizon.systems', '333', '[RSS]Cassandre', '#Genshiken', 'Shuuta!~Shuuta@Rizon-82AFB46A.ipv6.abo.wanadoo.fr', '1486578119']
[':irc.rizon.systems', '353', '[RSS]Cassandre', '=', '#Genshiken', ':[RSS]Cassandre', '@YamashitaRen', '@Tintalle', '@cavallone', '@Shuuta', 'Nala_Alan', 'Alexandre1er', 'Usavich', 'Muse-Kami-Sama', '@AnimeHQ', '+[XDCC]Genshiken|NEW']
[':irc.rizon.systems', '366', '[RSS]Cassandre', '#Genshiken', ':End', 'of', '/NAMES', 'list.']

['PING', ':irc.rizon.systems']
Responded to PING

[':YamashitaRen!~Yamashita@Rizon-58DBFC6.kimsufi.com', 'PRIVMSG', '#Genshiken', ':k']
```

Everytime the console shows "Responded to PING", rss2irc flood the channel with the entry message.

The chan is #Genshiken on irc.rizon.net/

**Comment by areebbeigh** [comment]
Ah, it was a silly mistake. The problem's fixed now. Thanks for the report. Feel free to open a new issue if you find any other problems.

**Comment by YamashitaRen** [comment]
Doesn't seem fixed unforunately :/
```
[':YamashitaRen!~Yamashita@Rizon-58DBFC6.kimsufi.com', 'PRIVMSG', '#Genshiken', ':!feed']

['PING', ':irc.uworld.se']
Responded to PING
[Channel Check] Joining #Genshiken

[':Kaccinko!~Kaccinko@Rizon-1161C759.fbx.proxad.net', 'JOIN', ':#Genshiken']
Joined to: set()
Kicked from: set()
[Channel Check] Joining #Genshiken

[':YamashitaRen!~Yamashita@Rizon-58DBFC6.kimsufi.com', 'PRIVMSG', '#Genshiken', ':rofl']
[Channel Check] Joining #Genshiken

['PING', ':irc.uworld.se']
Responded to PING
[Channel Check] Joining #Genshiken
```

I guess the bot send the message every "[Channel Check] Joining #Genshiken"...
